vagrant_support/checksetup.yml

```diff
@@ -18,7 +20,8 @@
     owner: vagrant
     group: vagrant
     mode: 0644
-
+  when: LAZY == 0
+  
 - name: run checksetup
   become: false
   shell: sg apache -c '/usr/local/bin/bmo-checksetup'
```

Comment:
To save even more time, we should pass --no-templates to checksetup.
It's probably safe to always do this.